multimedia/welle.io/Portfile

```diff
@@ -56,11 +56,47 @@ variant airspy description {Add Airspy support} {
     depends_lib-append port:airspy
 }
 
+variant cli description {Also compile welle-cli} {
+    # mpg123 is licensed as LGPL-2.1 except for 
+    # file src/libout123/modules/coreaudio.c which is GPL-2
+    # welle-cli relies on libout123 while the GUI does not.
+    # In order to have binaries distributed for welle-io,
+    # we enable only welle-gui by default. The user can enable
+    # cli variant to build also welle-cli locally.
+
+    configure.pre_args-replace -DBUILD_WELLE_CLI=OFF -DBUILD_WELLE_CLI=ON
+
+    license_noconflict-delete   mpg123
+
+    post-patch {
+        reinplace "s!\"index.html\"!\"${applications_dir}/welle-io.app/Contents/Resources/welle-cli/index.html\"!" \
+        ${worksrcpath}/src/welle-cli/webradiointerface.cpp
+        reinplace "s!\"index.js\"!\"${applications_dir}/welle-io.app/Contents/Resources/welle-cli/index.js\"!" \
+        ${worksrcpath}/src/welle-cli/webradiointerface.cpp
+    }
+
+    post-destroot {
+        ln -s ${applications_dir}/welle-io.app/Contents/MacOS/welle-cli \
+            ${destroot}${prefix}/bin/welle-cli
+    }
+}
+
+variant profiling description {Enable profiling (see README.md)} {
+    configure.pre_args-append \
+        -DPROFILING=ON
+}
+
+variant kiss_fft description {Use KISS FFT instead of FFTW} {
+    configure.pre_args-append \
+        -DKISS_FFT=ON
+}
+
```

Comment:
Shouldn't this remove the need for the dependency on `fftw-3-single`?

Reply:
Oh... yes, you're right !